mmv1/products/appengine/FlexibleAppVersion.yaml

```diff
@@ -139,6 +139,16 @@ properties:
         description: |
           List of ports, or port pairs, to forward from the virtual machine to the application container.
         item_type: Api::Type::String
+      - !ruby/object:Api::Type::Enum
+        name: 'instanceIpMode'
```

Comment:
We'll want to use this field in at least one test! You can modify https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go (and ideally we'd switch values between the two steps in update, to ensure that works)

Reply:
Done